galgebra/ga.py

```diff
@@ -617,8 +615,19 @@ def dop(self, *args, **kwargs):
         Instanciate and return a multivector differential operator for
         this, 'self', geometric algebra.
         """
-        kwargs['ga'] = self
-        return mv.Dop(*args, **kwargs)
+        return mv.Dop(*args, ga=self, **kwargs)
+
+    def pdop(self, *args, **kwargs):
+        """ Shorthand to construct a :class:`~galgebra.mv.Pdop` for this algebra """
+        return mv.Pdop(*args, ga=self, **kwargs)
+
+    def dop(self, *args, **kwargs):
+        """ Shorthand to construct a :class:`~galgebra.mv.Dop` for this algebra """
+        return mv.Dop(*args, ga=self, **kwargs)
+
+    def sdop(self, *args, **kwargs):
+        """ Shorthand to construct a :class:`~galgebra.mv.Sdop` for this algebra """
+        return mv.Sdop(*args, ga=self, **kwargs)
```

Comment:
Looks like these were intended to be the answer to the removal of `setGa` in #163